Sources/MessagingInApp/Views/InAppMessageView.swift

```diff
@@ -65,5 +71,36 @@ public class InAppMessageView: UIView {
         //      }
     }
 
-    private func displayInAppMessage(_ message: Message) {}
+    private func displayInAppMessage(_ message: Message) {
+        // There might already be a message displayed. If so, remove it and cleanup the resources.
+        // First, remove the subview that the inline manager has a reference to.
+        // Lastly, cleanup the inline manager.
+        subviews.first?.removeFromSuperview()
+        inlineMessageManager = nil
+
+        // Create a new manager for this new message to display and then display the manager's WebView.
+        let newInlineMessageManager = InlineMessageManager(siteId: gist.siteId, message: message)
+        newInlineMessageManager.startLoadingMessage()
+
```

Comment:
shouldn't we preload the new message before removing the first one to avoid visual delays?

Reply:
🤔 Not a bad idea. 

If a message is being shown on the device, the message expires, and then we want to show a new in-app message, I think this idea would be possible since we can wait a few seconds before we remove the expired message. 

But for the scenario where a close button is clicked and we want to show another message, I am not sure we want to pre-render the next message. If we never show the next message, that pre-render could be wasteful. I think this would be considered to be a future enhancement, agree?